variant_calling_benchmarks/common.py

```diff
@@ -0,0 +1,39 @@
+import pandas
+
+def extract_loci_string(patient, variant_filenames):
+    '''
+    Given a patient and a list of variant CSV files, return a string
+    like "chr1:2-10,chr2-50-51" that gives the loci of all variants
+    containined in the variant files.
+
+    Parameters
+    -----------
+    patient : string
+
+    variant_filesnames : list of string
+
+    Returns
+    -----------
+    string
+
+    '''
+    loci = []
+    for filename in variant_filenames:
+        df = pandas.read_csv(filename)
+        df = df.ix[df.patient == patient]
```

Comment:
Ah yea I mean the second one, I pretty much exclusively use that. Good to know. The pandas boolean indexing docs at least seem to suggest just passing a boolean mask: http://pandas.pydata.org/pandas-docs/stable/indexing.html#boolean-indexing